src/lib/entities/order_detail_product.py

```diff
@@ -0,0 +1,12 @@
+from src.utils.utils import equals
+
+
+class OrderDetailProduct:
+    def __init__(self):
```

Comment:
I think this entity needs to be related to the order?

Reply:
It is, but through **order_detail** entity. the **order_detail_product** is relate to the **order_detail** entity and the **order_detail** is relate to the order.